web-console/knikubevirt/vm-details/Resources/Resources.md

```diff
@@ -0,0 +1,40 @@
+# VM details - Devices-and-resources
+
+### Virtual machine
+
+## From VM details page
+![VM details - Overview tab - default](img/VM-2-0.jpg)
+Within the VM details > overview tab > overview section, a click on the ‘Devices & resources’ field will pop up the correlated modal.
+
+![Devices & resources modal - default](img/VM-3-0.png)
+The ‘Devices & resources’ modal currently contains only the ‘workload isolation’ option. This may expand in the future. The modal is titled this way, instead of  ‘Workload isolation’, in order to align with the implementation on the pod’s side.
+
+A sub-text below the title mentions that the scheduler will attempt to achieve the following settings. In case it is not possible, the VM will be scheduled in the best way it can (the same thing applies for pods, where this modal has more settings).
+
+The user selects ‘Isolate this workload…’ checkbox.
+
+![VM details - Overview tab - workload isolation enabled](img/VM-4-0.jpg)
+The preview on the ‘Devices & resources’ us updated to reflect the changes.
+
+## From VM creation wizard
+![Create VM - Advanced - Virtual hardware](img/Create-VM-4-2-00.png)
```

Comment:
Can we align this with the current design seen here:
https://codepen.io/matthewcarleton/pen/NWWZMZJ
Note the indent for each section (CD-ROMs, Display) under virtual hardware. As this section grows I think it will be useful to indent so scanning is simpler.

Reply:
I thought PF4 dropped the indent. Totally agree we should use it

Reply:
agreed, updated with spacing instead. 
https://codepen.io/matthewcarleton/pen/NWWZMZJ?editors=1100